frontend/src/components/programsVacancies/ManageProgramsVacancies.vue

```diff
@@ -0,0 +1,349 @@
+            </v-radio-group>
+          </div>
+
+          <div>
+            <p class="mb-2">
+              * Does your program have capacity to provide care to children who require additional support due to
+              developmental delay or disability (i.e. support for physical, cognitive, communicative, social, emotional,
+              and/or behavioral needs)?
+            </p>
+            <v-radio-group v-model="form.accommodatesSpecialNeeds" :disabled="!isEditing">
+              <v-row>
+                <v-col cols="auto"><v-radio label="Yes" :value="true" /></v-col>
+                <v-col cols="auto"><v-radio label="No" :value="false" /></v-col>
+              </v-row>
+            </v-radio-group>
+          </div>
+
+          <div>
+            <p class="mb-2">
+              * Do one or more of the child care staff (including yourself) have an Early Childhood Educator (ECE)
+              Certificate?
+            </p>
+            <v-radio-group v-model="form.ece" :disabled="!isEditing">
+              <v-row>
+                <v-col cols="auto"><v-radio label="Yes" :value="true" /></v-col>
+                <v-col cols="auto"><v-radio label="No" :value="false" /></v-col>
+              </v-row>
+            </v-radio-group>
+          </div>
+
+          <div>
+            <p class="mb-2">* Is the BC Early Learning Framework (ELF) used as a resource here?</p>
+            <v-radio-group v-model="form.elf" :disabled="!isEditing">
+              <v-row>
+                <v-col cols="auto"><v-radio label="Yes" :value="true" /></v-col>
+                <v-col cols="auto"><v-radio label="No" :value="false" /></v-col>
+              </v-row>
+            </v-radio-group>
+          </div>
+        </v-card>
+      </v-row>
+    </v-skeleton-loader>
+  </v-container>
+</template>
+
+<script>
+import AppButton from '@/components/guiComponents/AppButton.vue';
+
+import alertMixin from '@/mixins/alertMixin.js';
+import globalMixin from '@/mixins/globalMixin.js';
+
+import ProgramsVacanciesService from '@/services/programsVacancies.js';
+
+import {
+  AGE_GROUPS,
+  DAYS_OPTIONS,
+  EMPTY_PLACEHOLDER,
+  INDIGENOUS_LED_OPTIONS,
+  LANGUAGE_OPTIONS,
+  MEAL_OPTIONS,
+  PICKUP_OPTIONS,
+  PRESCHOOL_OPTIONS,
+  VACANCY_FIELDS,
+} from '@/utils/constants.js';
+import { formatUTCDate } from '@/utils/format';
+import rules from '@/utils/rules.js';
+
+export default {
+  name: 'ManageProgramsVacancies',
+  components: { AppButton },
+  mixins: [alertMixin, globalMixin],
+  data() {
+    return {
+      isLoading: false,
+      isEditing: false,
+      isProcessing: false,
+      programVacancies: null,
+      form: {},
+      ageGroups: AGE_GROUPS,
+      daysOptions: DAYS_OPTIONS,
+      languageOptions: LANGUAGE_OPTIONS,
+      indigenousLedOptions: INDIGENOUS_LED_OPTIONS,
+      preschoolOptions: PRESCHOOL_OPTIONS,
+      mealOptions: MEAL_OPTIONS,
+      pickupOptions: PICKUP_OPTIONS,
+      vacancyFields: VACANCY_FIELDS,
```

Comment:
If these are just constants, then we really don't want them in data as anything in data becomes reactive and there's overhead to that. Our standard is to assign them in the created() method (e.g. this.FOO = FOO) so that they can be used in the template.